Source/i_main.c

```diff
@@ -149,13 +155,17 @@ void I_SafeExit(int rc)
 
   // Run through all exit functions
 
-  while ((entry = exit_funcs))
+  for (; exit_priority < exit_priority_max; ++exit_priority)
   {
-    exit_funcs = exit_funcs->next;
-
-    if (rc == 0 || entry->run_on_error)
+    while ((entry = exit_funcs[exit_priority]))
     {
-      entry->func();
+      exit_funcs[exit_priority] = exit_funcs[exit_priority]->next;
+
+      if (rc == 0 || entry->run_on_error)
+      {
+//      fprintf(stderr, "Exit Sequence[%d]: %s (%d)\n", exit_priority, entry->name, rc);
```

Comment:
Have you forgotten about it?

Reply:
😄 No, I just don't want to spam stderr on exit. This is for debugging only.